<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleMapWindowBase</class>
 <widget class="QMainWindow" name="SimpleMapWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frameMap">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="mpActionZoomIn"/>
    <addaction name="mpActionZoomOut"/>
    <addaction name="mpActionPan"/>
    <addaction name="mpActionAddLayer"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMap"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWigTOC">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>42</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Таблица содержания</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="mpActionZoomIn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/ico/mActionZoomIn.png</normaloff>:/ico/mActionZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="mpActionZoomOut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/ico/mActionZoomOut.png</normaloff>:/ico/mActionZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="mpActionPan">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/ico/mActionPan.png</normaloff>:/ico/mActionPan.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="mpActionAddLayer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/ico/mActionAddLayer.png</normaloff>:/ico/mActionAddLayer.png</iconset>
   </property>
   <property name="text">
    <string>Add Layer</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
